CHANGELOG-PENDING.md

```diff
@@ -0,0 +1,21 @@
+# Unreleased Changes
```

Comment:
same comment as in the app, I don't think having a pending change log is worth it here.